### DISCUSSION TITLE
Set Icon rotate speed ?

### DISCUSSION THREAD

**Discussion by www516717402** [apache/airflow#34043] [Q&A]
How do I change the speed at which the icon in the upper left corner rotates? And Replace other images?

**Comment by potiuk** [comment]
You can't. It's not "customizable".  But if you wish, we welcome to add Pull Request to make it customizable. We have recently merged a change where we applied some modifications to it and make it adapt to latest standards for "Responsive Design on Motion"  https://github.com/apache/airflow/pull/34020  - so that it is disabled for people who prefer reduced motion and limited spinning time to 1.5 s  - so you can take inspiration from that.

**Comment by www516717402** [comment]
Thank you for reply.
